packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -139,7 +143,9 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
         return matchingStep.label;
       }
     }
-    return localValue.toString();
+    // For continuous steps default to showing 2 decimals in tooltip
+    // Consider making it configurable via a property
+    return Number(localValue.toFixed(2)).toString();
```

Comment:
To piggy pack off of Matt's comment, I think we want to make this same update to the `onChangeHandler` function. Either that, or in the `handleThumbMove` function we should trim the continuous value on line 221 to use 2 trailing digits.

Reply:
Fixed.